projects/dotcms-field-elements/src/components/dot-textfield/dot-textfield.tsx

```diff
@@ -18,17 +19,26 @@ import {
 })
 export class DotTextfieldComponent {
     @Element() el: HTMLElement;
-    @Prop() disabled = false;
-    @Prop() hint: string;
-    @Prop() label: string;
-    @Prop() name: string;
-    @Prop() placeholder: string;
-    @Prop() regexCheck: string;
-    @Prop() required: boolean;
-    @Prop() requiredMessage: string;
+    @Prop({ mutable: true })
+    disabled = false;
+    @Prop() hint = '';
+    @Prop({ mutable: true })
+    label = '';
+    @Prop() name = '';
+    @Prop() placeholder = '';
+    @Prop({ mutable: true })
+    regexCheck = '';
+    @Prop() required = false;
+    @Prop() requiredMessage = '';
     @Prop() type = 'text';
-    @Prop() validationMessage: string;
-    @Prop({ mutable: true }) value: string;
+    @Prop() validationMessage = '';
+    @Prop({ mutable: true })
+    value = '';
+
+    fieldAttr = {
```

Comment:
See comments in https://github.com/dotCMS/core-web/pull/1021/files#diff-15356a60e5d00893978145ac20b662e5